scripts/ci-unit-tests.ts

```diff
@@ -40,6 +46,7 @@ const SKIP_TEST_PACKAGES: Record<NodeVersion, VersionConfig> = {
       '@sentry/astro',
       '@sentry/nuxt',
       '@sentry/nestjs',
+      '@sentry-internal/eslint-plugin-sdk',
```

Comment:
A micro optimization here is to change `eslint-plugin-sdk` to only run test for whatever the node version of the local env is.

We can effectively treat it as a browser test then.

Reply:
actually I guess we should technically run this in multiple node versions, as this is published? 🤔 and actually probably bump the min. node version of this to 16+ as it does not seem to work on 14... 😅 it's internal so I wouild not care about this _too_ much 🤔 WDYT?